Sources/App/Controllers/User/UserController.swift

```diff
@@ -5,12 +5,10 @@ import Multipart
 
 final class UserController {
   
-  let drop: Droplet!
   let photoConroller: PhotoController
   
   init(drop: Droplet) {
-    self.drop = drop
-    self.photoConroller = PhotoController(drop: self.drop)
+    self.photoConroller = PhotoController(drop: drop)
```

Comment:
remove `self.` and you have a typo in `self.photoConroller`

Reply:
yes, thank you!